### PR TITLE
Small convenience upgrade to surgery + tweaks

### PR DESCRIPTION
**Pull request by florkeger**
**What does this PR do:**
This PR gives the current surgery rooms a very minor redesign. I moved certain objects into better angles for faster caretaking of patients while also making sure that surgery tools are now readied and tidely sorted in a bag by default. I also relocated the buttons for easier access during surgery and (probably) made a tiny bit more room with two gaps.

**Images of sprite/map changes (IF APPLICABLE):**
Sign: https://i.imgur.com/qCC8HLK.png
Locker: https://i.imgur.com/wIluWXC.png
Surgery: https://i.imgur.com/Jn6WISh.png

**Changelog:**

:cl: kazboo
add:  Added a surgeon locker with a recolored locker sprite. It's only filled with the neccesseties for surgeons.
add: Added 2 loaded ready-to-go wall mounted defibs to both ORs.
add: Added minor decals to borth ORs.
add: Firelocks added to both ORs.
add: Firelocks added to brig phys bay.
tweak: Adjusted outer OR windows, wiring and piping.
tweak: Small change to OR hallway flooring.
tweak: Body scanners moved behind OR table for better access; moved the buttons for same effect.
tweak: Tweaked the cameras in Captain Office and Vault. Both come now equipped with motion sensors by default.

imageadd: added a surgeon locker with recolored sprite
imageadd: added a new poster to both medbay ORs (wash your hands)
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/bottle.dm [hunk 1]
```diff
@@ -151,6 +151,12 @@
 	icon_state = "round_bottle"
 	list_reagents = list("sulfonal" = 30)
 
+/obj/item/reagent_containers/glass/bottle/antiviral
+	name = "Spaceacillin Bottle"
+	desc = "A small bottle containing a commonly used medical antidote."
```
Comment (TDSSS) [inline code/modules/reagents/reagent_containers/bottle.dm]: I'm not sure antidote is the right word here. You're not using it to neutralize a poison.

Comment (florkeger) [reply code/modules/reagents/reagent_containers/bottle.dm]: Ooh, right, what would it be instead?

Comment (TDSSS) [reply code/modules/reagents/reagent_containers/bottle.dm]: Maybe "A small bottle containing a commonly used medicine." ?

Comment (florkeger) [reply code/modules/reagents/reagent_containers/bottle.dm]: Hm that could work I guess. I'll change that up and await some more feedback.

Comment (florkeger) [reply code/modules/reagents/reagent_containers/bottle.dm]: Well something went terribly wrong. Apparently I have over 100k removes which is more than insanely unrealistic. Gotta have to look into that.

Comment (TDSSS) [reply code/modules/reagents/reagent_containers/bottle.dm]: Did you run the mapmerge scripts?

**Pull request closed without merging**